client/state/console-dispatch/index.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Console dispatcher Redux store enhancer
+ *
+ * @blame dmsnell
```

Comment:
I know we tend to annotate with author in wpcom, but I feel like these get out of date and I check blame anyway.